components/mixins/profile/profile-privacy.schema.json

```diff
@@ -18,12 +18,12 @@
       "properties": {
         "xdm:optOutConsentLevel": {
           "type": "object",
-          "title": "Opt-out consent level",
-          "description": "Opt-out consent level.",
+          "title": "Opt-out consent level (deprecated)",
+          "description": "Opt-out consent level (deprecated).",
```

Comment:
If this is to be deprecated, then add the "meta:status" : "deprecated" attribute here. This will make it so the field no longer appears in the UI (or shouldn't).